tests/primer/packages_to_prime.json

```diff
@@ -28,6 +28,9 @@
   "music21": {
     "branch": "master",
     "directories": ["music21"],
+    "pylint_additional_args": [
+      "--disable=invalid-name,bad-docstring-quotes,docstring-first-line-empty"
```

Comment:
I work on this package, so I know it doesn't use pylint for invalid-name. The additional f-string inferences results must just be swamping the output. It uses single quotes for docstrings, too. We don't need thousands of duplicative messages slowing things down.

Reply:
Isn't a project using single quote the only way to test thé Bad docstrings' quote message ?

Reply:
I wrote that comment (and pushed this change) before verifying it was the cause of the failed run. Perhaps it's not.

I will likely remove it.

But to your point, something so simple can be tested with a unit test.

Reply:
Sure, maybe this check is simple enough for a unit test (didn't check the code but it's likely). But the point of the primers is to encounter and test on a lot of unexpected code constructs to detect crashes without having to think about a relevant unit test first. If this check is simple/generic enough, why not disable it also for all primer repositories ? (For 'double quote docstrings repo' where it's never going to raise in particular).